webpack/ForemanRhCloudFills.js

```diff
@@ -33,10 +33,11 @@ const fills = [
   {
     slot: 'host-details-page-tabs',
     name: 'CVEs',
-    component: props => <CVEsHostDetailsTab {...props} />,
+    component: props => <CVEsHostDetailsTabWrapper {...props} />,
     weight: 300,
     metadata: {
       hideTab: vulnerabilityDisabled,
```

Comment:
The tab is not showing for me. How do I debug it?

Reply:
@jeremylenz ^ :pray:

Reply:
From Shim: "We are adding it to API, specifically to a single rabl. The only thing that needs to exist, is the insights facet. If the facet is not there, we won't add the node. Now the facet should be created on each package upload. Maybe in Viliam's case the host was there before my package upload pr got in, so he doesn't have the insights facet on the host. The easiest way would be to sync hosts with a button. I think I have enabled this use case, and it will properly sync the ids"

Reply:
I did a resycn with "Generate and upload report" but some hosts still don't show it. Interesting fact is that navigating from CVE counts form the hosts list table gets the tab shown, but it doesn't provide it a subscription/system id.